### PR TITLE
Small changes to certain commands + Command addition

### PR DESCRIPTION
**Pull request by Jan200101**
- Changed update command message to not say that streaming doesnt work
- added TWLoader command
- togglechannel now is easier to handle
- small text changes from #175
- made netinfo command a embed
- added 2 memes

### CODE REVIEW

**Comment by joo456** [comment]
dank may mays :ok_hand:

**Comment by NoNormies** [comment]
yes yes yes very good PR

File: addons/memes.py [hunk 1]
```diff
@@ -223,7 +223,18 @@ def __init__(self, bot):
...
+
+    @commands.command(pass_context=True, hidden=True)
+    async def helpers2(self, ctx):
+        """you are asking for help?"""
+        await self._meme(ctx, "http://i.imgur.com/O0okmlW.png")
+
+    @commands.command(pass_context=True, hidden=True, aliases=['sandwich'])
+    async def sandwichhax(self, ctx):
+        """the most avanced hacking method"""
+        await self._meme(ctx, "http://i.imgur.com/4v5Hz5d.jpg"
```
Comment (ihaveamac) [inline addons/memes.py]: there's a syntax error here.

**Comment by Jan200101** [comment]
fixed

**Comment by Jan200101** [comment]
This is no longer a total Meme PR

**Comment by ihaveamac** [comment]
this will be manually merged due to conflicts, if that's fine with you.

**Comment by Jan200101** [comment]
I made some changes and resolved the Conflicts.
You can view the changes in my latest Commit [9cb381c](https://github.com/916253/Kurisu/pull/170/commits/9cb381c0c2e1361b868cfca42ff7388f8e52371d)

I removed my additions to the blacklisted words and names of piracy software

**Comment by Jan200101** [comment]
Im actually going to write a addition now to the togglechannel command to allow easy additions

**Comment by Jan200101** [comment]
Togglechannel has been rewriten and the result is subjective.
The bot has been written for only one server and it being easier to use may or may not be useful

**Comment by Jan200101** [comment]
Rewrite of togglechannel is shit and the rest of the cog isnt needed.
Closing

**Pull request closed without merging**